src/LinQuadOptInterface.jl

```diff
@@ -113,7 +113,8 @@ ConstraintMapping() = ConstraintMapping(
 This function updates all of the references in `m`
 after we have deleted row `row` in the affine constraint matrix.
 """
-function shift_references_after_delete_affine!(m, row)
+shift_references_after_delete_affine!(m, row) = shift_references_after_delete_affine_base!(m, row)
+function shift_references_after_delete_affine_base!(m, row)
```

Comment:
Can you add a comment about why this is needed. This confused me until I went and looked at your Xpress branch.

Reply:
I will remove this!